apps/logging/tests/audit_logger_schemas.py

```diff
@@ -352,40 +554,89 @@ def get_pre_fetch_fhir_log_entry_schema(version):
         "start_time": {"type": "string"},
         "elapsed": {"type": "number"},
     },
-    "required": ["type", "uuid", "path", "sub", "code", "size", "start_time", "elapsed"]
+    "required": [
+        "type",
+        "uuid",
+        "path",
+        "sub",
+        "code",
+        "size",
+        "start_time",
+        "elapsed",
+    ],
 }
 
-'''
+"""
   Log entry schema used for tests in apps/logging/tests/test_loggers_management_command.py
-'''
+"""
 GLOBAL_STATE_METRICS_LOG_SCHEMA = {

```

Comment:
START-CODE-FOR-REVIEW-SECTION

In this file, only the `GLOBAL_STATE_METRICS_LOG_SCHEMA` and `GLOBAL_STATE_METRICS_PER_APP_LOG_SCHEMA` JSON schema were updated. These are used to validate JSON logging.